From 5c822696b07f455f797b085e18c01ab68e74ce15 Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Fri, 30 Jul 2021 21:12:02 +0000
Subject: [PATCH] Remove allconfigurations flag

Including BuildAllConfigurations build with Windows specific
TFMs as well, which don't build on source-build.
---
 eng/SourceBuild.props | 1 -
 1 file changed, 1 deletion(-)

diff --git a/eng/SourceBuild.props b/eng/SourceBuild.props
index 7dfc80d41fd..c7b40253a05 100644
--- a/eng/SourceBuild.props
+++ b/eng/SourceBuild.props
@@ -37,7 +37,6 @@
       <InnerBuildArgs>$(InnerBuildArgs) --arch $(TargetRidPlatform)</InnerBuildArgs>
       <InnerBuildArgs>$(InnerBuildArgs) --configuration $(Configuration)</InnerBuildArgs>
       <InnerBuildArgs>$(InnerBuildArgs) --ci</InnerBuildArgs>
-      <InnerBuildArgs>$(InnerBuildArgs) --allconfigurations</InnerBuildArgs>
       <InnerBuildArgs>$(InnerBuildArgs) --verbosity $(LogVerbosity)</InnerBuildArgs>
       <InnerBuildArgs>$(InnerBuildArgs) --nodereuse false</InnerBuildArgs>
       <InnerBuildArgs>$(InnerBuildArgs) --warnAsError false</InnerBuildArgs>
-- 
2.31.1

